examples/question_answering/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "question_answering"
+version = "0.1.0"
+description = "Example Kolena integration for question answering"
+authors = ["Kolena Engineering <eng@kolena.io>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.10"
+kolena = ">=0.82.0,<1"
+s3fs = "^2022.7.1"
+scikit-learn = "^1.1.2"
+numba = "^0.56.0"
```

Comment:
oh do we need `numba` and `scikit-learn`?

Reply:
oops, nope!